contracts/contracts/UserReplicaSetManager.sol

```diff
@@ -0,0 +1,137 @@
+pragma solidity ^0.5.0;
+
+import "./interface/RegistryInterface.sol";
+import "./registry/RegistryContract.sol";
+import "./interface/UserFactoryInterface.sol";
+import "./SigningLogic.sol";
+
+
+/** @title Contract for Audius user replica sets */
+contract UserReplicaSetManager is RegistryContract, SigningLogic {
+
+    RegistryInterface private registry = RegistryInterface(0);
+    bytes32 private userFactoryRegistryKey;
+    address deployer;
+
+    // spID to ServiceProvider delegateOwnerWallet
+    mapping (uint => address) spIdToCreatorNode;
+
+    // Struct used to represent replica sets
+    struct ReplicaSet {
+        uint primary;
+        uint[] secondaries;
+    }
+
+    // Current userId to replica set
+    mapping (uint => ReplicaSet) artistReplicaSets;
+
+    constructor(
+        address _registryAddress,
+        bytes32 _userFactoryRegistryKey,
+        uint _networkId
+    ) SigningLogic("User Replica Set Manager", "1", _networkId) public
+    {
+        require(
+            _registryAddress != address(0x00) &&
+            _userFactoryRegistryKey.length != 0,
+            "requires non-zero _registryAddress"
+        );
+
+        registry = RegistryInterface(_registryAddress);
+        userFactoryRegistryKey = _userFactoryRegistryKey;
+        deployer = msg.sender;
+    }
+
+    // Called from dataOwnerWallet value as msg.sender
+    // TODO: Add signature for relay - signer must be the target delegateOwnerWallet for this replicaId
+    function registerCreatorNode(uint _spID, address _delegateOwnerWallet) external {
```

Comment:
eliminate special case for register, instead use specialAddress override to updateCreatorNode for initial seeding and security